tests/system/src/test/java/test/robot/javafx/scene/NodeInitializationBackgroundThreadTest.java

```diff
@@ -727,8 +728,16 @@ private void accessRegion(Region c) {
     }
 
     private <T extends Node> void test(Supplier<T> generator, Consumer<T> operation) {
-        T visibleNode = generator.get();
-        String title = visibleNode.getClass().getSimpleName();
+        AtomicReference<T> ref = new AtomicReference();
+        runAndWait(() -> {
```

Comment:
What was the rationale for this change? This means we are no longer testing the ability to construct a node off thread.

Related to this, I think a separate set of tests where we construct instances of the node in a loop would be useful, in addition to the set of tests that mutates the node in a loop.

Reply:
I think this change is why the Tooltip test is no longer failing. The failure mode reported in [JDK-8348100](https://bugs.openjdk.org/browse/JDK-8348100) happens when  constructing Tooltip on a background thread.

Reply:
The very first (visible) node should be created in the FX application thread (something I missed earlier).  All other instances are created in the background thread - notice the second call to `generator.get()` inside the background thread lambda.

Reply:
> Related to this, I think a separate set of tests where we construct instances of the node in a loop would be useful, in addition to the set of tests that mutates the node in a loop.

this is a good idea!  thank you.